src/api/entities/Subsidy/__tests__/index.ts

```diff
@@ -88,6 +89,84 @@ describe('Subsidy class', () => {
     });
   });
 
+  describe('method: setAllowance', () => {
+    afterAll(() => {
+      sinon.restore();
+    });
+
+    test('should prepare the setAllowance procedure with the correct arguments and context, and return the resulting transaction queue', async () => {
```

Comment:
I think we want to use `it` now, considering this is going into alpha

Reply:
This looks like its still `test` instead of `it`

Reply:
My bad. The file somehow didn't save. Pushed up a small change for this